src/lib/fragments/AgGrid.react.js

```diff
@@ -66,6 +69,39 @@ const xssMessage = (context) => {
 
 const NO_CONVERT_PROPS = [...PASSTHRU_PROPS, ...PROPS_NOT_FOR_AG_GRID];
 
+const agGridRefs = {};
+
+const eventBus = {
+    listeners: {},
+    on(id, data, callback) {
+        if (!(id in eventBus.listeners)) {
+            eventBus.listeners[id] = {};
+        }
+        eventBus.listeners[id][data] = callback;
+    },
+    dispatch(data) {
+        for (const id in eventBus.listeners) {
+            if (data in eventBus.listeners[id]) {
+                eventBus.listeners[id][data]();
```

Comment:
To make it clearer what's going on here (`id` is listening to events from `data` which is also an ID), let's replace `data` with `targetId`.
```suggestion
    on(id, targetId, callback) {
        if (!(id in eventBus.listeners)) {
            eventBus.listeners[id] = {};
        }
        eventBus.listeners[id][targetId] = callback;
    },
    dispatch(targetId) {
        for (const id in eventBus.listeners) {
            if (data in eventBus.listeners[id]) {
                eventBus.listeners[id][targetId]();
```